net-dns/c_ares/c_ares-1.14.0.recipe

```diff
@@ -0,0 +1,85 @@
+SUMMARY="A C library for asynchronous DNS requests"
+DESCRIPTION="This is c-ares, an asynchronous resolver library. It is intended \
+for applications which need to perform DNS queries without blocking, or need \
+to perform multiple DNS queries in parallel. The primary examples of such \
+applications are servers which communicate with multiple clients and programs \
+with graphical user interfaces."
+HOMEPAGE="https://c-ares.haxx.se/"
+COPYRIGHT="	2004-2018 by Daniel Stenberg et al
+	1998-2013 by the Massachusetts Institute of Technology
+	1987-2001 The Regents of the University of California
+	1996-2004 by Internet Software Consortium
+	2009 by Jakub Hrozek
+	2010 Jeremy Lal
+	2012 Marko Kreen
+	2005 Dominick Meglio"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://c-ares.haxx.se/download/c-ares-$portVersion.tar.gz"
+CHECKSUM_SHA256="45d3c1fd29263ceec2afc8ff9cd06d5f8f889636eb4e80ce3cc7f0eaf7aadc6e"
+SOURCE_DIR="c-ares-$portVersion"
+PATCHES="c_ares-$portVersion.patchset"
+
+ARCHITECTURES="?x86 !x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="?x86 !x86_gcc2"
+
+PROVIDES="
+	c_ares$secondaryArchSuffix = $portVersion compat >= 1
+	lib:libcares$secondaryArchSuffix = 2.2.0 compat >= 2
+	"
+if [ -z "$secondaryArchSuffix" ]; then
+	PROVIDES="$PROVIDES
+		cmd:ahost$secondaryArchSuffix = $portVersion compat >= 1
+		cmd:adig$secondaryArchSuffix = $portVersion compat >= 1
+		cmd:acountry$secondaryArchSuffix = $portVersion compat >= 1
+		"
+fi
+
```

Comment:
Is there a reason why the commands are not included for secondaryArchSuffix (as primary (x86_gcc2) is broken)?

Reply:
Good catch.
For `c_ares-1.10.0.recipe` that's OK because it builds (or at least is supposed to build) on all primary arches.

But for the newer releases, we should not drop the commands when `$secondaryArchSuffix` is not empty.